lib/workers/repository/config-migration/pr/index.spec.ts

```diff
@@ -169,6 +169,40 @@ describe('workers/repository/config-migration/pr/index', () => {
       expect(platform.createPr.mock.calls[0][0].prBody).toMatchSnapshot();
     });
 
+    it('creates non-semantic PR title', async () => {
+      await ensureConfigMigrationPr(
+        {
+          ...config,
+          prHeader: '\r\r\nThis should not be the first line of the PR',
+          prFooter:
+            'There should be several empty lines at the end of the PR\r\n\n\n',
+        },
+        migratedData
+      );
+      expect(platform.createPr).toHaveBeenCalledTimes(1);
+      expect(platform.createPr.mock.calls[0][0].prTitle).toBe(
+        'Migrate config renovate.json'
+      );
+    });
+
+    it('creates semantic PR title', async () => {
+      await ensureConfigMigrationPr(
+        {
+          ...config,
+          commitMessagePrefix: '',
+          semanticCommits: 'enabled',
+          prHeader: '\r\r\nThis should not be the first line of the PR',
+          prFooter:
+            'There should be several empty lines at the end of the PR\r\n\n\n',
+        },
+        migratedData
+      );
+      expect(platform.createPr).toHaveBeenCalledTimes(1);
+      expect(platform.createPr.mock.calls[0][0].prTitle).toBe(
+        'chore(config): migrate config renovate.json'
```

Comment:
```suggestion
        'chore(config): migrate renovate.json'
```

The scope already says `chore(config)`. How about dropping the second `config`?

Reply:
it depends if semantic vommits are enabled. so i would leave as is for now